### ISSUE TITLE
Forbes.com videos blocked 

### ISSUE THREAD

**Issue by dechamp** [easylist/easylist#18313]
#### Description
We are migrating to Connatix for our video service, and we are seeing that our videos are being blocked for `elements.video`.  We play videos on the main url Forbes.com and any sub-domains.

We do not have a production link at this time as we are in development but we have great concerns of our timeline and this affecting. 

#### List the website(s) you're having issues with:
```
- Forbes.com

```

#### What happens?
Our videos are being blocked due to the domain being targeted.

#### Problem Filter
```
||elements.video^$third-party
```
#### Screenshot(s) 
![Pasted_Image_16_01_24__15_21 (1)](https://github.com/easylist/easylist/assets/4752199/ecaf37a9-f571-4594-a9cf-4c000dd7954f)

#### Filter lists you're using:
EasyList through adblocker

**Comment by partingscientist** [comment]
@Yuki2718 Filter mentioned no longer exists.

**Comment by Yuki2718** [comment]
Yeah, removed by https://github.com/easylist/easylist/commit/6a0d60a233379697cf86c5a91dabc006156f0801
